bitnami/thanos/Chart.yaml

```diff
@@ -15,4 +15,4 @@ maintainers:
 name: thanos
 sources:
   - https://github.com/bitnami/bitnami-docker-thanos
-version: 0.4.3
+version: 0.4.4
```

Comment:
Transforming the Objstore into a secret might break backwards incompatibility. 
Please test upgrading from previous version to the one including your changes to ensure it works smoothly. If that's not the case, we might need to bump the major version to include this change.

Reply:
Upgraded to major release, we are removing a capability now and won't work with older values.yaml.